packages/director/src/execution/utils.ts

```diff
@@ -9,7 +9,9 @@ export const getClaimedSpecs = (run: Run, groupId: string) =>
 export const getFirstUnclaimedSpec = (run: Run, groupId: string) =>
   getSpecsForGroup(run, groupId).find((s) => !s.claimedAt);
 export const getFailedSpecs = (run: Run, groupId: string) =>
-  getSpecsForGroup(run, groupId).filter((s) => s.results!.stats.failures > 0);
+  getSpecsForGroup(run, groupId).filter(
```

Comment:
Technically `skipped` are also failed, cypress just isn't running those tests and marks them as skipped.
```ts
getSpecsForGroup(run, groupId).filter((s) =>
    s.results ? s.results.stats.failures + s.results.stats.skipped > 0 : false
  );
```